tests/template_tests/filter_tests/test_truncatewords_html.py

```diff
@@ -24,15 +24,15 @@ def test_truncate2(self):
             truncatewords_html(
                 '<p>one <a href="#">two - three <br>four</a> five</p>', 4
             ),
-            '<p>one <a href="#">two - three …</a></p>',
+            '<p>one <a href="#">two - three <br> …</a></p>',
```

Comment:
Perhaps unfortunate, especially given it's a `<br>`, but likely more strictly correct. Otherwise, not sure if we can avoid trailing void elements just before the replacement?